src/model/transformations/collected-variable.js

```diff
@@ -207,6 +208,21 @@ function getCollectedScope(questionsLoop, id, componentsStore) {
   })
   return isfound;
 }
+
+function getTableDynamique(componentsStore, id) {
+  let tableId = "";
+  Object.values(componentsStore).filter(components => 
+    components.type === QUESTION && 
+    components.responseFormat.type === "TABLE" && 
```

Comment:
We have constants somewhere for the TABLE and LIST strings

Reply:
@EmmanuelDemey  je viens de faire un Push